src/mon/AuthMonitor.cc

```diff
@@ -1870,6 +1891,80 @@ bool AuthMonitor::prepare_command(MonOpRequestRef op)
   return false;
 }
 
+/* Generate the caps that should be present in the entity's auth keyring
+ * after running the "fs authorize" command. This is done by merging the
+ * caps already present in the client's auth keyring with the new caps
+ * provided by the user at "fs authorize" command.
+ */
+int AuthMonitor::_gen_wanted_caps(EntityAuth& e_auth,
+  vector<string>& newcaps, ostream* out)
+{
+  enum caps_update_reqd_states {
+  NOT_REQD,	// caps need not be updated
```

Comment:
How about:

```
CAP_UPDATE_NEEDED
CAP_UPDATE_SKIP
CAP_UPDATE_MISC_ERROR
```

Reply:
Better. But I am using `CAP_UPDATE_NOT_REQD` instead of `CAP_UPDATE_SKIP` since we are not skipping but it's  not needed.